entities/errors/error_group_wrapper_test.go

```diff
@@ -0,0 +1,56 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package errors
+
+import (
+	"bytes"
+	"os"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestName(t *testing.T) {
+	cases := []struct {
+		env string
+		set bool
+	}{
+		{env: "something", set: true},
+		{env: "something", set: false},
+		{env: "", set: true},
+		{env: "false", set: true},
```

Comment:
not sure I understand. Which of those cases should have recovery disabled?
can we have field for expected behaviour/output: `expectedDisabled: true` / `expectedDisabled: false`?

Reply:
if you set the env to "true" it does not recover and the test fails - but I haven't found a way to recover from the panic that I don't recover in the waitgroup and so the test just fails. Happy if you have a better idea

Reply:
ok, got it.
unfortunately I've no clue how to catch panic in other than main routine. does not seem to be even possible https://stackoverflow.com/questions/50409011/handling-panics-in-go-routines.